meeko/__init__.py

```diff
@@ -50,6 +50,15 @@
 from .openff_xml_parser import get_openff_epsilon_sigma
 from .hydrate import Hydrate
 
+import logging
+from rdkit import rdBase
+rdkit_logger = logging.getLogger("rdkit")
+rdkit_logger.handlers[0].setLevel("WARNING")
+rdkit_logger.handlers[0].setFormatter(
+    logging.Formatter('[RDKit] %(levelname)s:%(message)s'),
```

Comment:
Because it's in init, it will change the format of RDKit logging messages (severity >= warning) after meeko is imported. Do you want to enforce this?

Reply:
Just leaving a record of what we discussed: I don't know. It may be useful to know which messages come from RDKit.